src/views/components/task-view/task-view.js

```diff
@@ -22,12 +19,13 @@ import i18n from '../../../i18n.js';
 import {notificationActions} from '../../../notification';
 import { TakeOwnershipModal }  from '../take-ownership-modal';
 import TaskCreator from "../task-creator/task-creator";
-
+import _ from 'lodash';
```

Comment:
I've read the importing lodash may cause a big package size so it's better to import only the parts that we need.
Not sure if that is the correct case here
https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark/